s3/replication/common/src/s3replicationcommon/s3_range_read.py

```diff
@@ -0,0 +1,134 @@
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+import aiohttp
+import sys
+from s3replicationcommon.aws_v4_signer import AWSV4Signer
+from s3replicationcommon.log import fmt_reqid_log
+from s3replicationcommon.s3_common import S3RequestState
+from s3replicationcommon.timer import Timer
+
+
+class S3AsyncGetRangeRead:
```

Comment:
I did not get it here. Why do we need another GetRangeRead? We already have GetObject.  

GetObject(BucketName, ObjectName, offset=0, length=0)

range is offset,offset+length

Reply:
Done. Update the code of GetObject.